lib/csvable.rb

```diff
@@ -16,7 +16,7 @@ def from_array_of_hashes(data = [], humanize = true)
           .map(&:to_s)
       end
 
-      CSV.generate do |csv|
+      CSV.generate({:col_sep => sep}) do |csv|
```

Comment:
preferred style is `{ col_sep: sep }` instead of `{ :col_sep => sep }`. 
No need to worry about it now though, when we run Rubocop on the entire codebase it will auto-change these for us.